CMakeLists.txt

```diff
@@ -300,6 +300,7 @@ message("-- ${White}${PROJECT_NAME} -- Link Libraries: ${LINK_LIBRARY_LIST}${Col
 target_link_libraries(${PROJECT_NAME} ${LINK_LIBRARY_LIST})
 set_target_properties(${PROJECT_NAME} PROPERTIES POSITION_INDEPENDENT_CODE ON)
 set_target_properties(${PROJECT_NAME} PROPERTIES LINKER_LANGUAGE CXX)
+target_link_libraries(${PROJECT_NAME} ${PROJECT_SOURCE_DIR}/libs/third_party/ffts/libffts.a)
```

Comment:
@r-abishek : I think we can't do this since source of this library is unknown and someone can possibly put a security hack here. Can you please replace this with
https://github.com/mreineck/pocketfft
It only uses header file and hence no source or so is required. You can just have the header file replicated in thirdparty with their license intact

Reply:
@rrawther Yes that's true, and to that end, we'll check both these options:
- if a drop-in replacement with the above lib header works in terms of getting the same audio training convergence
- or if all necessary dependencies from the ".a" file could be extracted and added in RPP as raw C/C++ headers and built along with RPP.